.github/workflows/memory_store.yml

```diff
@@ -0,0 +1,114 @@
+      REGION:
+        required: false
+        type: string
+        default: us-central1
+        
+    secrets:
+      SA_CREDENTIALS:
+        required: true
+        description: The Service Account that will be used to perform changes on GCP
+      PROJECT_ID:
+        required: true
+        description: The GCP project ID
+
+    outputs:
+      url: 
+        description: The URL of this specific Cloud Run revision
+        value: ${{ jobs.deploy-cloud-run-service.outputs.url }}
+
+jobs:
+  provision-memory-store-cluster:
+    defaults:
+      run:
+        working-directory: ${{inputs.WORK_DIR}}
+    name: Provisioning ${{inputs.SERVICE}}
+    runs-on: ubuntu-latest
+    outputs:
+      url: ${{ steps.deploy.outputs.url }}
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - id: 'auth'
+        uses: 'google-github-actions/auth@v0'
+        with:
+          credentials_json: '${{ secrets.SA_CREDENTIALS }}'
+          create_credentials_file: true
+          export_environment_variables: true
+      
+      - name: Setup Cloud SDK
+        uses: google-github-actions/setup-gcloud@v0
+
+      - name: Make sure jq in installed
+        run: sudo apt-get install jq
+
+      - name: Fix env vars
+        run: |
+          echo "SHORT_SHA=`echo ${GITHUB_SHA} | cut -c1-8`" >> $GITHUB_ENV
+
+
+      - name: Provision Cluster
+        id: deploy
+        run: |-
+          emptyResponse=$(echo "[]")
+          output=$(gcloud redis instances list --filter="projects/${{ secrets.PROJECT_ID }}/locations/${{ inputs.REGION }}/instances/${{ inputs.CLUSTER_NAME }}" --format=json --region=${{ inputs.REGION }})
+
+          if [ $output == $emptyResponse ]; then
+            echo "Provisioning new MemoryStore cluster ${{ inputs.CLUSTER_NAME }}"
+            gcloud services enable redis.googleapis.com --project=${{ secrets.PROJECT_ID }}
+
+            gcloud redis instances create ${{ inputs.CLUSTER_NAME }} \
+            --tier=standard \
+            --size=${{ inputs.MEMORY }} \
+            --region=${{ inputs.REGION }} \
+            --read-replicas-mode=READ_REPLICAS_ENABLED \
+            --replica-count=${{ inputs.REPLICA_COUNT }} \
+            --redis-version=redis_6_x \
+            --network=default \
+            --display-name=${{ inputs.CLUSTER_NAME }} \
+            --rdb-snapshot-period=${{ inputs.SNAPSHOT_PERIOD }} \
+            --connect-mode=direct-peering  
+          else
+            gcloud redis instances update ${{ inputs.CLUSTER_NAME }} --region=${{ inputs.REGION }} \
+            --rdb-snapshot-period=${{ inputs.SNAPSHOT_PERIOD }} \
+            --replica-count=${{ inputs.REPLICA_COUNT }} \
+            --size=${{ inputs.MEMORY }}
+          fi
+          
+          echo "::set-output name=url::$(echo $clusterDesc | jq -r '"\(.readEndpoint):\(.readEndpointPort)"')"
```

Comment:
Is `jq` part of github action?

Reply:
I am not sure, but I can change to `value`